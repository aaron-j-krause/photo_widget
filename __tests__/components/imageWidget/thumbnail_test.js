import React from 'react'
import renderer from 'react-test-renderer'

import Thumbnail from '../../../src/components/imageWidget/thumbnail'

test('should render correctly', () => {
  const testProps = { description: 'test_description', url: 'test_url' }
  const thumbnail = renderer.create(<Thumbnail {...testProps} />)
  expect(thumbnail).toMatchSnapshot()
})
