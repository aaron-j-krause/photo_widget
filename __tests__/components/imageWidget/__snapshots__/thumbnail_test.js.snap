exports[`test should render correctly 1`] = `
<img
  alt="test_description"
  className={undefined}
  src="test_url" />
`;
